xml/obs_build_config.xml

```diff
@@ -193,6 +193,9 @@ Keep: debbuild</screen>
            <para>Configure usage of ccache for build of packages mentioned by VALUE. Ccache package will be automatically
                installed and configured. The directory <filename>/.ccache/</filename> will be configured as cache dir.
                To configure ccache, the file <filename>/.ccache/ccache.conf</filename> can be modified as part of the build process by the <envar>$BUILD_USER</envar> environment variable. </para>
+
+           <para>if ccache is enabled but there is no archive present (eg: in case of newly branched packages, wiped binaries etc).
```

Comment:
```suggestion
           <para>If ccache is enabled, but there is no archive present (e.g.: in case of newly branched packages, wiped binaries etc...).
```

Reply:
I would suggest to replace "e.g." with "for example".